src/languages.js

```diff
@@ -30,6 +30,10 @@ Languages.get = function(code, key, callback) {
 	var languageData;
 
 	fs.readFile(path.join(__dirname, '../public/language/', code, key), { encoding: 'utf-8' }, function(err, data) {
+		if (err) {
```

Comment:
Good idea.

I’ve added an additional check to make sure the callback is only called with `err` when `err.code !== 'ENOENT'`.